packages/@sanity/migrate/src/fetch-utils/fetchStream.ts

```diff
@@ -8,17 +8,23 @@ export interface HTTPError extends Error {
   statusCode: number
 }
 
-export function assert2xx(res: Response) {
+export async function assert2xx(res: Response): Promise<void> {
   if (res.status < 200 || res.status > 299) {
-    const err = new Error(`HTTP Error ${res.status}: ${res.statusText}`) as HTTPError
+    const response = await res.json()
```

Comment:
The json parsing may fail here, should we handle this specifically?

Reply:
that was my first pass with try catch but since it already catches it somewhere else it broke other things but I think I can inline a catch and lot it and it should still throw the error correctly